packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -997,6 +1010,11 @@ export class StatsAnalyzer extends EventsScope {
       this.statsResults[mediaType][sendrecvType].headerBytesSent = result.headerBytesSent;
       this.statsResults[mediaType][sendrecvType].retransmittedBytesSent =
         result.retransmittedBytesSent;
+      this.statsResults[mediaType][sendrecvType].isRequested =
+        result.isRequested ||
+        (result.lastRequestedUpdateTimestamp &&
+          performance.timeOrigin + performance.now() - result.lastRequestedUpdateTimestamp <
+            MQA_INTERVAL);
```

Comment:
This doesn't feel like the right place to put this calculation. The problem with putting this calculation here is that `processOutboundRTPResult` is called every 5 seconds, so this would not tell us whether the stream was requested within the last interval, but rather whether the stream was requested within the last 60 seconds, which could include the previous interval.

Technically, at the time the MQE is sent, this may end up being correct because the last 60 seconds at that point would be the within the last interval. But I think it would be better to just store the `isRequested` and `lastRequestedUpdateTimestamp` values here, and only do this calculation every 60 seconds when we send the MQE out.

Anna's PR does something similar to this, so you may use it as a reference: https://github.com/webex/webex-js-sdk/pull/3651

Reply:
I agree with @brycetham. It's a great point. With that logic, it should be calculated every 60 seconds, not 5.